### PR TITLE
Add repairs get documentation

### PR DESCRIPTION
**Pull request by ZaidYounan**
[Associated PR](https://github.com/easy2use/gearbox/pull/3598)

### CODE REVIEW

File: endpoints/repairs.md [hunk 1]
```diff
@@ -1,8 +1,150 @@
...
+- purchase_order
+- location
+- created_at
+- updated_at
+- site
+- repair_items.repair_types
+- repairers.name
+- repairers.code
+- repair_items.assigned_user_id
+- repair_items.general_ledger_code_id
```
Comment (Gotenks-J) [inline endpoints/repairs.md]: These need to be updated to match the current filterable attributes

File: endpoints/repairs.md [hunk 2]
```diff
@@ -1,8 +1,150 @@
...
+```
+URL: https://api.gearbox.com.au/public/v1/repairs
+Method: GET
+Authorization: Bearer $ACCESS_TOKEN
+```
+
+### 200 - Successful response
+
+```JSON
+{"repairs": [
```
Comment (Gotenks-J) [inline endpoints/repairs.md]: This sample response will need to be updated to match the actual response from the API

File: endpoints/repairs.md [hunk 3]
```diff
@@ -1,8 +1,146 @@
...
+    repair_items.status,
+    repair_items.priority,
+    repairers.rep_name,
+    repairers.code,
+    repair_items.assigned_user,
+    repair_items.general_ledger_code,
+    parts.part_number,
+    parts.quantity,
+    parts.each,
+    parts.total
```
Comment (JDrizzy) [inline endpoints/repairs.md]:
```suggestion
    repair_items.repairers.name,
    repair_items.repairers.code,
    repair_items.assigned_user,
    repair_items.general_ledger_code,
    repair_items.parts.part_number,
    repair_items.parts.quantity,
    repair_items.parts.each,
    repair_items.parts.total
```

File: endpoints/repairs.md [hunk 4]
```diff
@@ -1,8 +1,146 @@
...
+    hours_open,
+    closed,
+    date_closed,
+    hours_closed,
+    odometer_closed,
+    purchase_order,
+    location,
+    created_at,
+    site,
+    repair_items.repair_types,
```
Comment (JDrizzy) [inline endpoints/repairs.md]:
```suggestion
    repair_items.repair_types.type,
```

**Pull request merged**